sonic_y_cable/y_cable_base.py

```diff
@@ -262,8 +265,8 @@ def get_switch_count(self, port, count_type):
                  an Integer, the actual physical port connected to a Y cable
             count_type:
                  a string, for getting the count type
```

Comment:
Suggest changing to `One of the following predefined constants:`

Reply:
fixed